src/Build/Definition/ProjectCollection.cs

```diff
@@ -881,20 +861,8 @@ internal int MaxNodeCount
         /// - So that the owner of this project collection can force the XML to be loaded again
         /// from disk, by doing <see cref="UnloadAllProjects"/>.
         /// </summary>
-        internal ProjectRootElementCache ProjectRootElementCache
-        {
-            get
-            {
-                // no locks required because this field is only set in the constructor.
-                return _projectRootElementCache;
-            }
-
-            private set
-            {
-                // no locks required because this field is only set in the constructor.
-                _projectRootElementCache = value;
-            }
-        }
+        // No locks required because this field is only set in the constructor.
```

Comment:
Since it's an autoprop with no setter, you can delete this comment

Reply:
Done